tests/unit/customizations/s3/test_fileinfo.py

```diff
@@ -26,13 +25,38 @@
 from awscli.customizations.s3.fileinfo import TaskInfo
 
 
-class TestSaveFile(unittest.TestCase):
+class BaseMD5Test(unittest.TestCase):
```

Comment:
We typically only create a shared base class if it's going to be shared my multiple tests, but I only see `TestSaveFile` subclassing from it.  Was there a reason to create this base class?